Build/Tasks/BuildPackages.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+namespace DotNetNuke.Build.Tasks
+{
+    using Cake.Frosting;
+
+    /// <summary>A cake task to compile the platform and create all of the packages.</summary>
+    /// <remarks>This is the task run during CI.</remarks>
+    [IsDependentOn(typeof(CleanArtifacts))]
+    [IsDependentOn(typeof(UpdateDnnManifests))]
+    [IsDependentOn(typeof(GenerateSecurityAnalyzerChecksums))]
+    [IsDependentOn(typeof(SetPackageVersions))]
+    [IsDependentOn(typeof(CreateInstall))]
+    [IsDependentOn(typeof(CreateUpgrade))]
+    [IsDependentOn(typeof(CreateNugetPackages))]
+    [IsDependentOn(typeof(GeneratePackagesChecksums))]
+    public sealed class BuildPackages : FrostingTask<Context>
```

Comment:
What is the purpose/goal of this task addition?  I'm concerned that this will be deleting a source controlled file.